doc/source/structures/vessels/engine.rst

```diff
@@ -44,6 +44,18 @@ Some of the Parts returned by :ref:`LIST PARTS <list command>` will be of type E
         * - :attr:`ISP`
           - scalar
           - `Specific impulse <isp>`_
+        * - :attr:`VACUUMISP`
+          - scalar
+          - `Vacuum Specific impulse <isp>`_
+        * - :attr:`VISP`
+          - scalar
+          - `Synonym for VACUUMISP`_
```

Comment:
When Johann gets back we should ask him about how to change the template so it can handle listing aliases.  We end up doing this thing a lot and it would be better to just name the aliases all in the same section.